src/demo/ui.jsx

```diff
@@ -751,7 +752,9 @@ let Loading = class Loading extends React.Component {
   render() {
     return (
       <Body>
-        <div style={styles.loading}>Loading...</div>
+        <div style={styles.loading}>
+          <img src={loadingGif} style={{maxWidth: 300}} />
```

Comment:
Should this be a %age for responsive displays?

Reply:
yeah, that sounds good. i updated it to 50% maxWidth

Reply:
Thanks.  To be finicky, I wonder if we should ensure that the loading bot is never larger than the actual bot shown in the tutorial?

Reply:
the bot sizing is done in different places, so i'm not sure we can 100% ensure that, but i see elsewhere that the bot is 30% so i'll set it to that instead